app/reporting-framework/json-reports/lung-cancer-treatment-monthly-summary-base.json

```diff
@@ -155,146 +193,153 @@
         },
         {
             "type": "derived_column",
-            "alias": "walk_in",
+            "alias": "enrolled_patients",
             "expressionType": "simple_expression",
             "expressionOptions": {
-            "expression": "(IF(cur_visit_type IN (2), 1, NULL))"
-                       }
+                "expression": "(IF ((program.date_completed IS NULL AND program.program_id IN(32)), 1, 0))"
+            }
         },
         {
             "type": "derived_column",
-            "alias": "referred_from_clinic",
+            "alias": "alive_patients",
```

Comment:
kindly rename this to **active_patients** patients. Patient not **dead** alone qualifies the patient as active. **Active** would make more sense if you associate with an active program